pandas/tools/plotting.py

```diff
@@ -359,7 +360,16 @@ def scatter_matrix(frame, alpha=0.5, figsize=None, ax=None, grid=False,
     density_kwds = density_kwds or {}
 
     # workaround because `c='b'` is hardcoded in matplotlibs scatter method
-    kwds.setdefault('c', plt.rcParams['patch.facecolor'])
+    # only set default for `c` if `color` kwarg not supplied
+    # edgecolors default required otherwise `color` has a border
+    # GH 14855
+    kwds.setdefault('edgecolors', 'none')
+    if 'color' in kwds:
+        pass
+    if 'facecolor' in kwds:
+        kwds.setdefault('facecolor', plt.rcParams['patch.facecolor'])
+    if 'c' in kwds:
+        kwds.setdefault('c', plt.rcParams['patch.facecolor'])
```

Comment:
I do not understand this, if `'c'` is in kwds, then `setdefault` is a no-op.

You probably also want to version guard this, mpl 2.0 scatter respects the color cycle
http://matplotlib.org/2.0.0rc1/users/dflt_style_changes.html#scatter